mkdocs.yml

```diff
@@ -6,6 +6,7 @@ nav:
   - About: about.md
   - Contribute: contribute.md
   - Resources: resources.md
+  - Code of conduct: code_of_conduct.md
```

Comment:
Happy with the PR to go in with how things are, but perhaps it makes sense to have the code of conduct immediately after contribute, also keeps resources and cite us (two bib related tabs) together.

Reply:
Good point! Please go ahead and adjust the order